README.md

```diff
@@ -684,6 +684,11 @@ allocations by setting the CuPy CUDA allocator to
 >>> cupy.cuda.set_allocator(rmm.rmm_cupy_allocator)
 ```
 
+
+**Note:** This only configures CuPy to use the current RMM resource for allocations. 
+It does not initialize nor change the current resource, e.g., enabling a memory pool. 
```

Comment:
```suggestion
It does not initialize or change the current resource; for example, enabling a memory pool. 
```

Reply:
I don't think either of these nits need be picked. This was really just meant to be a very quick PR. 

**nor** - used as a function word to introduce the second or last member or the second and each following member of a series of items each of which is negated.

...*not* initialize *nor* change...

Seems right to me. 

And I just prefer `e.g.`.